.github/workflows/ci.yml

```
@@ -39,28 +48,30 @@ jobs:
             ghc: 8.2.2
     steps:
     - uses: actions/checkout@v2
-      if: github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.ref == 'refs/heads/main'
 
-    - uses: actions/setup-haskell@v1.1.4
+    - uses: haskell/actions/setup@v1
       id: setup-haskell-cabal
       name: Setup Haskell
       with:
         ghc-version: ${{ matrix.ghc }}
         cabal-version: ${{ matrix.cabal }}
 
+    - name: Configure
+      run: |
+        cabal configure --enable-tests --enable-benchmarks --enable-documentation --test-show-details=direct --write-ghc-environment-files=always
+
     - name: Freeze
       run: |
         cabal freeze
 
-    - uses: actions/cache@v2.1.4
+    - uses: actions/cache@v2.1.3
```

Heh, I guess `relude` wasn't using the latest version of all actions...
```suggestion
    - uses: actions/cache@v2.1.4
```